From b2091f6d75ee10928b12d115dc3286e479f303d5 Mon Sep 17 00:00:00 2001
From: Nicolas Goaziou <mail@nicolasgoaziou.fr>
Date: Thu, 30 Jun 2016 15:20:51 +0200
Subject: [PATCH 32/50] Move obsolete declarations into "org-compat.el"

* lisp/org-agenda.el (org-agenda-list):
(org-agenda-goto-today):
(org-agenda-reset-view): Remove reference to obsolete
`org-agenda-ndays'.
---
 lisp/ox-odt.el | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lisp/ox-odt.el b/lisp/ox-odt.el
index 0bb1dfd6d..29a0b8573 100644
--- a/lisp/ox-odt.el
+++ b/lisp/ox-odt.el
@@ -281,7 +281,6 @@ except that the foreground and background colors are set
 according to the default face identified by the `htmlfontify'.")
 
 (defvar hfy-optimizations)
-(define-obsolete-variable-alias 'hfy-optimisations 'hfy-optimizations "25.1")
 (defvar org-odt-embedded-formulas-count 0)
 (defvar org-odt-embedded-images-count 0)
 (defvar org-odt-image-size-probe-method
-- 
2.13.2

